packages/pilot/src/containers/RecipientDetails/Config/style.css

```
@@ -0,0 +1,47 @@
+@import "former-kit-skin-pagarme/dist/styles/spacing.css";
+@import "former-kit-skin-pagarme/dist/styles/colors.css";
+
+.paddingTop > div {
+  padding: 24px 0 0;
```

Opa mais uma variável que poderia vir do `former-kit-skin`
```suggestion
  padding: var(--spacing-large) 0 0;
```